### PR TITLE
Update items.json

### PR DESCRIPTION
**Pull request by Psychotic10**

### CODE REVIEW

**Comment by keiracodes** [comment]
proud of u

File: data/items.json [hunk 1]
```diff
@@ -842,7 +842,7 @@
         "name": "lucky 7",
         "aliases": ["7"],
         "emoji": "<:nypsi_7:1001553113368506538>",
-        "description": "improved chances for slots",
+        "description": "improved chances for slots 65%",
```
Comment (tekoh) [inline data/items.json:845]: whats this for

**Comment by keiracodes** [comment]
LMFAO

File: data/items.json (hunk 1 above)
Comment (Psychotic10) [reply data/items.json:845]: No reason lol sorry ok mb mb u keep that normal

**Pull request closed without merging**